src/DynamoCoreWpf/ViewModels/Preview/InfoBubbleViewModel.cs

```diff
@@ -465,6 +465,32 @@ public void OnRequestAction(InfoBubbleEventArgs e)
         /// </summary>
         /// <param name="dynamoViewModel"></param>
         public InfoBubbleViewModel(DynamoViewModel dynamoViewModel)
+        {
+            InitializeInfoBubble(dynamoViewModel);
+        }
+
+        /// <summary>
+        /// This constructor will update the TopBottom and BottomRight positions of the warnings since the beginning.
+        /// </summary>
+        /// <param name="dynamoViewModel"></param>
+        /// <param name="nodeViewModel">This parameter will be used to get the TopBottom and BottomRight positions of the NodeModel</param>
+        public InfoBubbleViewModel(DynamoViewModel dynamoViewModel, NodeViewModel nodeViewModel)
```

Comment:
`DynamoViewModel` was made public property on NodeViewModel so this function can be simplified to accept only one param

Reply:
@QilongTang fixed in the next commit: https://github.com/DynamoDS/Dynamo/pull/13015/commits/ef41a1d4192da420f31d4dcdd78d26ef955e77ef